pkg/pfcpsim/session/far_builder.go

```diff
@@ -91,17 +101,20 @@ func (b *farBuilder) BuildFAR() *ie.IE {
 
 	if b.zeroBasedOuterHeader {
 		fwdParams.Add(ie.NewOuterHeaderCreation(S_TAG, 0, "0.0.0.0", "", 0, 0, 0))
-	} else {
+	} else if b.teid != 0 && b.downlinkIP != "" {
```

Comment:
Had to add this if otherwise we would get the outerheadercreation in any case.
I'll revisit the code to avoid this